cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+    "baseURL": "http://localhost:3000"
```

Comment:
This should probably be 3001 since you will be working with a Rails based API

Reply:
Updated to 3001